### ISSUE TITLE
TCP/IP Attack Lab - SYN Flooding only successfull without previous TCP connection

### ISSUE THREAD

**Issue by CSendner** [seed-labs/seed-labs#62]
In the TCP/IP Attack Lab task 3.1, you're launching an SYN flooding attack. One observation you made was that the SYN flood was only successful if no previous TCP connections were made (not only telnet). This is due to a mitigation of the Kernel. The TCP/IP stack will reserve 1/4 of the backlog for "proven destinations" if SYN Cookies are disabled. We can flush those IPs on the server:

`sudo ip tcp_metrics flush`

Afterward, the attack will be successful again - without a reboot.

**Comment by kevin-w-du** [comment]
Thank you so much. I really appreciate it. Will update the lab description.

**Comment by kevin-w-du** [comment]
I added the explanation to lab description. When I tried the attack on my VM again together, the SYN flooding attack seems to be successful always, without flushing tcp_metrics or restarting the container. This was not the case two months ago. Something might have changed to the container image or my VM image that turned off such a mitigation method. If you have any insight on this, that will be very helpful. Thanks.
